amqp_broker.go

```diff
@@ -80,10 +93,15 @@ func NewAMQPCeleryBrokerByConnAndChannel(conn *amqp.Connection, channel *amqp.Ch
 	broker := &AMQPCeleryBroker{
 		Channel:    channel,
 		connection: conn,
-		exchange:   NewAMQPExchange("default"),
-		queue:      NewAMQPQueue("celery"),
+		exchange:   NewAMQPExchange(defaultExchangeName),
+		queue:      NewAMQPQueue(defaultQueueName),
 		rate:       4,
+		routes: map[string]*Route{"default": &Route{
```

Comment:
I wanted to make sure that each broker has its own default that can be modified independently, in case there are multiple brokers (see new commit with SetDefaultRoute).  If you like it better with a DefaultRoute variable at the module level we'd just need to make a deep copy for each instantiated broker